docs/VLSI/Building-A-Chip.rst

```diff
@@ -1,6 +1,50 @@
 .. _build-a-chip:
 
 Building A Chip
-==============================
+===============
+
+In this section, we will discuss many of the ASIC-specific transforms and methodologies within Chipyard.
+For the full documentation on how to use the VLSI tool flow, see the `Hammer Documentation <https://hammer-vlsi.readthedocs.io/>`__.
+
+Transforming the RTL
+--------------------
+Building a chip requires specializing the generic verilog emitted by FIRRTL to adhere to the constraints imposed by the technology used for fabrication.
+This includes mapping Chisel memories to available technology macros such as SRAMs, mapping the input and output of your chip to connect to technology IO cells, see :ref:`Barstools`.
+In addition to these required transformations it may also be benefical to transform the RTL to make physical design easier.
+This often includes modifying the logical hierarchy to match the physical hierarchy through grouping components together or flattening components into a single larger module.
+
+
+Modifying the logical hierarchy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Something about complex chips -> needs hierarchical
```

Comment:
I think a quick mention of the grouping pass would go far here. It's often the case that physical constraints imply a certain hierarchical flow is needed, which requires transforming the logical hierarchy, blah blah blah...

Reply:
I made a quick mention in the latest commit.
I think an example in chipyard would be good (e.g. a cluster?) but that will have to be a separate PR.

Reply:
We have examples of this in our configs area. Please refer to the documentation on how to create heterogeneous setups and further modify chipyard